src/edgeVariablesContainer.ts

```diff
@@ -44,11 +42,12 @@ export class MSPropertyContainer extends variables.BaseVariableContainer {
     }
 
     public async setValue(adapter: EdgeDebugAdapter, name: string, value: string): Promise<string> {
-        const msDebuggerPropertyId = this._childPropertiesMapping[name];
+        const msDebuggerPropertyId = this._childPropertiesMapping.get(name);
 
         if (!msDebuggerPropertyId) {
-            logger.error(`Cannot find msDebuggerPropertyId for {name}`);
-            throw coreUtils.errP(localize("edge.debug.error.notFoundMsDebuggerPropertyId", "Cannot find msDebuggerPropertyId for a property."));
+            // If msDebuggerPropertyId is not present, default to super setValue for this variable.
+            let result = await super.setValue(adapter, name, value);
```

Comment:
Is there a way to add a new localized error message at this point? I actually want to add it for the msSetDebuggerPropertyValue call below as it will throw an error if unsuccessful to ensure that users know if this failed unexpectedly

Reply:
Shouldn't we add it for both the cases when we fail to get the result?

For localized message you can follow the same convention we have in the codebase: localize(key, message) , where key should be unique.